string[] joints
---
uint8 SUCCEEDED = 0
uint8 NOT_FOUND = 0
uint8 FAILED = 0

uint8 result
string[] joints